### PR TITLE
Move dkan_migrate_base repo into dkan core

### PR DESCRIPTION
**Pull request by janette**
Issue: civic 3800

Moved to a consolidation branch
https://github.com/NuCivic/dkan/pull/1387

### CODE REVIEW

**Comment by janette** [comment]
currently failing on features/widgets.feature
```
And I wait and press "Save"                                                                    # Drupal\DKANExtension\Context\DKANContext::iWaitAndPress()
      Couldn't find button Save within 30 seconds (Exception)
```
locally that file fails on all **And I wait for "Customize this page"** so seems the 30 second time limit is not long enough for the page to complete itself

File: modules/dkan/dkan_migrate_base/README.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
+[![Build Status](https://circleci.com/gh/NuCivic/dkan_migrate_base.svg?branch=7.x-1.x&style=svg)](https://circleci.com/gh/NuCivic/dkan_migrate_base)
+
+DKAN "Harvester" using Migrate module
```
Comment (dafeder) [inline modules/dkan/dkan_migrate_base/README.md]: Need to change the title to explain difference between migrate base and harvester

File: modules/dkan/dkan_migrate_base/README.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
+[![Build Status](https://circleci.com/gh/NuCivic/dkan_migrate_base.svg?branch=7.x-1.x&style=svg)](https://circleci.com/gh/NuCivic/dkan_migrate_base)
```
Comment (dafeder) [inline modules/dkan/dkan_migrate_base/README.md]: Remove the build status

File: modules/dkan/dkan_migrate_base/dkan-module-init.sh [hunk 3]
```diff
@@ -0,0 +1,62 @@
+
```
Comment (dafeder) [inline modules/dkan/dkan_migrate_base/dkan-module-init.sh]: Remove entire file.

File: modules/dkan/dkan_migrate_base/composer.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (dafeder) [inline modules/dkan/dkan_migrate_base/composer.json]: Remove/move to root test folder

File: modules/dkan/dkan_migrate_base/dkan_migrate_base.make [hunk 5]
```diff
@@ -0,0 +1,6 @@
+core = 7.x
```
Comment (dafeder) [inline modules/dkan/dkan_migrate_base/dkan_migrate_base.make]: Remove makefile

File: modules/dkan/dkan_workflow/test/features/dkan_workflow.feature [hunk 6]
```diff
@@ -0,0 +1,542 @@
+@api @disablecaptcha @javascript
```
Comment (dafeder) [inline modules/dkan/dkan_workflow/test/features/dkan_workflow.feature]: Same question. Why in this PR?

File: modules/dkan/dkan_workflow/test/composer.json [hunk 7]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (dafeder) [inline modules/dkan/dkan_workflow/test/composer.json]: Why is this in this PR?

File: drupal-org.make [hunk 8]
```diff
@@ -8,6 +8,7 @@ includes:
   - "modules/dkan/dkan_workflow/dkan_workflow.make"
   - "modules/dkan/dkan_data_story/dkan_data_story.make"
   - "modules/dkan/dkan_topics/dkan_topics.make"
+  - "modules/dkan/dkan_migrate_base/dkan_migrate_base.make"
```
Comment (dafeder) [inline drupal-org.make]: Not needed

File: modules/dkan/dkan_migrate_base/.gitignore [hunk 9]
```diff
@@ -0,0 +1,3 @@
+composer.lock
```
Comment (dafeder) [inline modules/dkan/dkan_migrate_base/.gitignore]: None of these needed anymore; remove file

**Comment by dafeder** [comment]
We can maybe merge this with the `consolidation` branch - @janette to review

**Pull request closed without merging**